setup-machine.sh

```diff
@@ -282,11 +282,15 @@ sudo chmod -R u=rwX /var/vx/config
 sudo chmod -R g=rX /var/vx/config
 sudo chmod -R o-rwX /var/vx/config
 
+# set the flag to run basic configuration when the machine boots
+sudo touch /var/vx/config/RUN_BASIC_CONFIGURATION_ON_NEXT_BOOT
+
 # non-graphical login
 sudo systemctl set-default multi-user.target
 
 # setup auto login
 sudo mkdir -p /etc/systemd/system/getty@tty1.service.d
+sudo ln -s /vx/code/config/autologin.sh /vx/config/autologin.sh
```

Comment:
why this? Can't we just run it directly from `/vx/code/config/autologin.sh` ?

Reply:
it seemed like this was the pattern all throughout the setup-machine script, so I was just following it. also i saw a comment at the top that said /vx/code was read only - dunno if that's true.

Reply:
yes, it is read-only *after* we lock down the machine, which is why I think we want to put that script there, so it can't be modified later.